src/js/tabs/pid_tuning.js

```diff
@@ -2343,7 +2343,10 @@ TABS.pid_tuning.changeRatesType = function(rateTypeID) {
 
     if (self.currentRatesType !== RC_tuning.rates_type) {
         sameRatesType = false;
-        dialogRatesType.showModal();
+
+        if (!dialogRatesType.hasAttribute('open')) {
+            dialogRatesType.showModal();
+        }
```

Comment:
I think the button assignments below should also be avoided if the dialog is already open.

Reply:
Yep, you are right. Will fix now with a rebase and squash